pinot-common/pom.xml

```diff
@@ -46,6 +46,8 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration combine.children="override">
+          <forkCount>1</forkCount>
+          <reuseForks>true</reuseForks>
```

Comment:
Isn't this the default value? https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#reuseForks

Reply:
Parent pom disables fork reuse and most sub-projects inherit the setting.

Reply:
Why don't we change that in the parent pom then? If tehre is a project here reuse breaks tests, we should fix it in that specific pom

Reply:
I don't want to further increase the scope of the PR. I know that some projects to require it due to static init and others have flaky tests .